crates/miden-lib/src/account/interface/mod.rs

```diff
@@ -252,6 +258,13 @@ impl AccountInterface {
             matches!(component_interface, AccountComponentInterface::BasicFungibleFaucet(_))
         }) {
             basic_fungible_faucet.send_note_body(*self.id(), output_notes)
+        } else if let Some(_network_fungible_faucet) =
+            self.components().iter().find(|component_interface| {
+                matches!(component_interface, AccountComponentInterface::NetworkFungibleFaucet(_))
+            })
+        {
+            // Network fungible faucet doesn't support send_note_body for now
```

Comment:
```suggestion
            // Network fungible faucet doesn't support send_note_body, because minting
            // is done via a MINT note.
```

Nit: It's never going to support `send_note_body`, right? If so, I'd rephrase this.